pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[project]
+name = "superphot-plus"
+license = {file = "LICENSE"}
+readme = "README.md"
+authors = [
+    { name = "LF", email = "name@you.com" }
```

Comment:
I think this should be Kaylee's name and contact info (if she wants)

Reply:
Yes, I'm ok with this!